.gitignore

```diff
@@ -20,3 +20,5 @@ Build
 *.swp
 *.orig
 .arcconfig
+
+build.ninjae
```

Comment:
Is this meant to be `build.ninja`? If so it is already covered by a wildcard

Reply:
This is probably coming from the "sed -ie” commands in the script where “e” is taken as the name for the temp file.